minecraft/service/signaling/dial.go

```
@@ -0,0 +1,138 @@
+	}
+
+	opts := &websocket.DialOptions{
+		HTTPHeader: make(http.Header),
+		HTTPClient: d.HTTPClient,
+	}
+	opts.HTTPHeader.Set("Authorization", token.AuthorizationHeader)
+	requestURL := d.Environment.ServiceURI.JoinPath(
+		"/ws/v1.0/signaling",
+		d.NetworkID,
+	)
+	c, _, err := websocket.Dial(ctx, requestURL.String(), opts)
```

golangci-lint (2.12.2)

[error] response body must be closed

(bodyclose)